code/dataaccess/extractqueryparams.q

```diff
@@ -27,8 +27,8 @@ extractqueryparams:{[inputparams;queryparams]
 extracttablename:{[inputparams;queryparams]@[queryparams;`tablename;:;inputparams`tablename]};
 
 extractpartitionfilter:{[inputparams;queryparams]
-  //If an RDB return the partitionfilters as empty
-  if[`rdb~inputparams[`metainfo;`proctype];:@[queryparams;`partitionfilter;:;()]];
+  //If Intraday return the partitionfilters as empty
+  if[inputparams[`metainfo;`proctype] in `rdb`tr_seg1`tr_seg2;:@[queryparams;`partitionfilter;:;()]];
```

Comment:
I don't think proctypes should be hardcoded here (I know `rdb` was already, but I think that was probably the wrong thing to do)

Reply:
I agree, I think for the most part the hardcoding of proctypes will be addressed with Shawn's RDB proctype change PR #547 as we had realised it was very impractical to change so many scripts for scalability.  Some other changes along the same vein with relation to the tailsort are being planned also.